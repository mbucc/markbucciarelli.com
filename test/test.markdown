October 28, 2022

tags: baz, biz

Foo Bar Baz Title
===================

This paragraph has a margin
<?mn: note1?>
Old MacDonald had a note, eieio.
<?mn?>
note.


Second-level headers like this should be wrapped in a section tag.
--------------------------------------------------------

And images should be wrapped in a figure tag

![](/img/t.png)

But images that have a wrapper should not have a second one added.

<?fullwidth?>
![](/img/u.png)

