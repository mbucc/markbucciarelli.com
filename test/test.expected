October 28, 2022

tags: baz, biz

Foo Bar Baz Title
===================

This paragraph has a margin
<label for=note1 class=margin-toggle>&#8853;</label>
<input type=checkbox id=note1  class=margin-toggle />
<span class=marginnote>
Old MacDonald had a note, eieio.
</span>
note.


<section>
Second-level headers like this should be wrapped in a section tag.
--------------------------------------------------------

And images should be wrapped in a figure tag

<figure>
![](/img/t.png)
</figure>

But images that have a wrapper should not have a second one added.

<figure class=fullwidth>
![](/img/u.png)
</figure>

</section>
